### DISCUSSION TITLE
the shareable is not a valid DXO - expect content_type DXO but got None

### DISCUSSION THREAD

**Discussion by ajulyav** [NVIDIA/NVFlare#1760] [Q&A]
Hello,

What could be a reason of this error? NV version is 2.0.

Server stops just after starting the run. The dataset is located properly, got the link to the dataset too. 

Thanks!

**Comment by holgerroth** [comment]
Please share the server and client log files. If possible use the latest version 2.3.0.

**Comment by YuanTingHsieh** [comment]
There could be many reasons.
Like client side failed and send back empty result.
Like server side failed and thus empty result is processed by aggregators.

As Holger pointed out, we could help more if you can share the server and client logs.
The "app" or "job" that you are running.
The system OS and python version and lib versions.

**Reply by YuanTingHsieh** [reply]
@ajulyav ^
